arcgis-ios-sdk-samples/Content Display Logic/Controllers/ContentTableViewController.swift

```diff
@@ -249,6 +228,9 @@ extension ContentTableViewController: UISearchResultsUpdating {
             return
         }
         
+        // do not preserve cell expansion when loading new results
+        expandedRowIndexPaths = []
```

Comment:
I'm sorry for not noticing this the first time around.

```suggestion
        expandedRowIndexPaths.removeAll()
```